hackathon/mdakits/README.md

```diff
@@ -34,7 +34,57 @@ MDAKits paper
   TX, 2023. doi:
   [10.25080/gerudo-f2bc6f59-00a](https://doi.org/10.25080/gerudo-f2bc6f59-00a).
 
+## Getting started
+
+Pick an idea for a stand-alone feature you, or others, want to see 
+possible through MDAnalysis - see the list of Project ideas below!
+We encourage you to discuss any ideas in the lead-up to the hackathon, 
+here or on the UGM discord channel (More details TBA).
+
+Read our guide on making an MDAKit (https://mdakits.mdanalysis.org/makingakit.html).
+(Additional materials TBA.)
+
+Some other things to keep in mind:
+- *Use the MDAnalysis API*: MDAnalysis features a mature API aimed to
+promote extensibility. We recommend you see how our collection of base
+classes and other existing tools could simplify your Kit and present a
+consistent interface to potential users; e.g.  AnalysisBase for
+trajectory-based analysis
+https://userguide.mdanalysis.org/stable/examples/analysis/custom_trajectory_analysis.html
+- *Using external packages*: Many tools and analyses have already been
+implemented as stand-alone tools or in other packages. It’s worth
+checking to see what’s available - it may be easier to include an external
+tool as a dependency than to make a Kit “from scratch”. Keep in mind how
+easy it would be to install and move data between MDAnalysis and the
+external implementation, and make sure to check the licence
+before including it in your Kit.
```

Comment:
I'll leave this open in case we don't want to encourage this or licensing details change, but I think we probably don't want to rely on individual, new developers to understand licenses.

Reply:
There's no way licensing would change in the span of two weeks - there's 187 contributors we'd have to contact.

Reply:
You could add here that the Apache 2.0 license should be watched out for.

Reply:
```suggestion
before including it in your Kit.  For now, MDAKits must remain GPLv3+ compatible.
We recognise that licensing compatibility can be quite complicated and nuanced;
for example, [software licensed under Apache 2.0 can be included in MDAKits with
GPLv3+ licenses, but MDAnalysis code (GPLv3+) cannot be included in Apache
2.0 projects](https://www.apache.org/licenses/GPL-compatibility.html).
Therefore, we encourage you to ask any questions you might have about licenses
to a core developer.
```

Reply:
Is this similar to what you had in mind, @IAlibay?